src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -298,8 +302,9 @@ private ParseUserDefinitionResult ParseUDFsAndNamedFormulas(string script, Parse
 
                 if (_curs.TidCur == TokKind.Equ)
                 {
+                    var declaration = script.Substring(declarationStart, _curs.TokCur.Span.Min - declarationStart);

```

Comment:
>        var declaration = script.Substring(declarationStart, _curs.TokCur.Span.Min - declarationStart); [](http://example.com/codeflow?start=12&length=103)

Have you tried this with really large NF declarations to see how this scales?

Reply:
Tested with many UDFs that each had many parameters